### ISSUE TITLE
main page - empty

### ISSUE THREAD

**Issue by Darkslayer0905** [lzawadowicz1/Project-two-Website-Redesign#1]
Not sure if there is supposed to be any kind of image on the first page. Seems kind of empty under the menu bar even though you are able to scroll a little bit.

**Comment by lzawadowicz1** [comment]
Yeah...the only page at the moment that has any real content at the moment is the Classes & Events page.
